python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/monitor-sessions.j2

```diff
@@ -54,10 +54,19 @@
 {%             endif %}
 {%         endif %}
 {%     endfor %}
+{%     if monitor_sessions_default_encapsulation_gre.payload is arista.avd.defined %}
```

Comment:
as per current logic for this we have to define monitor_sessions then only it will check for monitor_sessions_default_encapsulation_gre.payload. But tried to update the logic still its not generating correct data when i do not define monitor_sessions